src/movie.c

```diff
@@ -35,28 +34,28 @@ movie_t* movie_new(unsigned int cols, unsigned int rows) {
     return movie;
 }
 
+// Intenta reservar un asiento en la sala.
 bool movie_reserve_seat(movie_t* m, unsigned int col, unsigned int row, int id) {
-    assert(m != NULL);
-
-    if (col >= m->ncols || row >= m->nrows) {
+    if (col >= m->ncols || row >= m->nrows)
```

Comment:
Y por qué quitas el assert? Vuelve a agregarlo y ponle corchetes al if
```suggestion
    assert(m != NULL);

    if (col >= m->ncols || row >= m->nrows) {
```